index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+   <head>
+<script src="script.js"></script>       
+
+   <body>
+       <h1>Web Calculator</h1>
+       <form>
+       var x <input type="text" id="varx" value=""/>
+       var y <input type="text" id="vary" value=""/>
+       <input type="button" style="background-color: darkorchid ;" onclick="sum();" value=" Submit"/>
+       <br><br>
+       var result <input type="text" id="varresult">
+   </form>
+       </body> 
+       </head>
+       </html> 
+
+ 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
You have a lot of unnecessary whitespace here. Moving forward, try to eliminate extra lines of whitespace such as this so as to keep your code file as succinct as possible.